app/javascript/components/sponsor_us/SponsorshipOption.jsx

```diff
@@ -2,23 +2,32 @@ import React from 'react';
 import PropTypes from 'prop-types';
 import Card from 'components/Card';
 
-const SponsorshipOption = ({ title, amount, selected }) => {
+const SponsorshipOption = ({ title, amount, selected, onClick }) => {
```

Comment:
All the changes I made in this file are in order to make the radio buttons properly accessible, using [this article](https://www.a11ywithlindsey.com/blog/create-custom-keyboard-accessible-radio-buttons) as guidance.